core/src/main/java/com/novoda/accessibility/ClosedCaptionManagerFactory.java

```diff
@@ -0,0 +1,63 @@
+package com.novoda.accessibility;
+
+import android.annotation.TargetApi;
+import android.content.Context;
+import android.os.Build;
+import android.view.accessibility.CaptioningManager;
+
+public class ClosedCaptionManagerFactory {
+
+    public static CaptionManager createCaptionManager(Context context) {
```

Comment:
Or `with` as some libraries seem to be doing, but that's usually for builders, so dunno.